openssl/src/crypto/pkey.rs

```diff
@@ -69,6 +70,22 @@ impl PKey {
         }
     }
 
+    /// Reads private key from PEM, takes ownership of handle
```

Comment:
Not sure what you mean, this function is already a method on `PKey` (`PKey::private_key_from_pem`).
And renaming it to `from_pem` would make it ambiguous because there could also be a function for reading public keys (`public_key_from_pem`)